packages/ncids-js/src/components/nci-header/__tests__/search/search.test.ts

```diff
@@ -137,7 +68,7 @@ describe('NCI Search', () => {
 
 		fireEvent.focus(inputArea);
 
-		expect(addEventListener.mock.calls).toHaveLength(3);
+		expect(addEventListener.mock.calls).toHaveLength(2);
 		userEvent.keyboard('cancer');
```

Comment:
i know you didn't touch this, but is line 72-75 working? `userEvent` and `waitFor` are missing `await` on the async function calls